mlflow/gateway/constants.py

```diff
@@ -0,0 +1 @@
+PROVIDERS = {"openai", "anthropic", "databricks_serving_endpoint", "mlflow_served_model"}
```

Comment:
This should be dinaymically created be from the `Provider` model.

Reply:
+1

Reply:
yeah it was a placeholder :)

Reply:
Filed #8712